### ISSUE TITLE
`wd clean!` inexplicably clearing all warp points

### ISSUE THREAD

**Issue by lschumm** [mfaerevaag/wd#41]
The Bug
======

For example, with no warp points set:
```
➜  ~
➜  ~ wd list
 * All warp points:
➜  ~ wd add test
 * Warp point added
➜  ~ wd list
 * All warp points:
test  ->  ~
➜  ~ cd Downloads/
➜  Downloads 
➜  Downloads w add test2
 * Warp point added
➜  Downloads wd list
 * All warp points:
 test  ->  ~
test2  ->  ~/Downloads
➜  Downloads wd clean!
 * Nonexistent directory: test -> ~
 * Nonexistent directory: test2 -> ~/Downloads
 * Cleanup complete. 2 warp point(s) removed
➜  Downloads wd list
 * All warp points:
```

Info
===

zsh 5.2 (x86_64-apple-darwin16.0). MacOS Sierra 10.12.1 (16B2657). MacBookAir7,1. ZSH selected from Terminal -> Preferences -> General -> Shells open with -> Command (complete path): /bin/zsh

**Comment by lschumm** [comment]
Resolved in PR #37. Please merge!

**Comment by altschuler** [comment]
Merged, thanks a lot for reporting and reminding :)
